crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -866,3 +878,21 @@ pub struct MultisafepayErrorResponse {
     pub error_code: i32,
     pub error_info: String,
 }
+
+fn get_error_response_if_failure(
+    error_code: Option<String>,
+    error_message: Option<String>,
+    http_code: u16,
+    connector_transaction_id: Option<String>,
+) -> types::ErrorResponse {
+    types::ErrorResponse {
+        code: error_code.unwrap_or(consts::NO_ERROR_CODE.to_string()),
+        message: error_message
+            .clone()
+            .unwrap_or(consts::NO_ERROR_MESSAGE.to_string()),
+        reason: error_message,
```

Comment:
Take both `reason` and `message` as argument, and construct the error response using those parameters.
Also we can move this functions to utils as this would be generic across connectors.
Consult with @ArjunKarthik once and then move it to utils